docs/getting-started-guides/rkt/index.md

```diff
@@ -10,25 +10,30 @@ We still have [a bunch of work](http://issue.k8s.io/8262) to do to make the expe
   *(Note that systemd is not required by rkt itself, we are using it here to monitor and manage the pods launched by kubelet.)*
 
 - Install the latest rkt release according to the instructions [here](https://github.com/coreos/rkt).
-  The minimum version required for now is [v0.8.0](https://github.com/coreos/rkt/releases/tag/v0.8.0).
+  The minimum version required is [v1.9.0](https://github.com/coreos/rkt/releases/tag/v1.9.0).
 
-- Note that for rkt version later than v0.7.0, `metadata service` is not required for running pods in private networks. So now rkt pods will not register the metadata service be default.
-
-- Since release [v1.2.0-alpha.5](https://github.com/kubernetes/kubernetes/releases/tag/v1.2.0-alpha.5),
-the [rkt API service](https://github.com/coreos/rkt/blob/master/api/v1alpha/README.md)
+The [rkt API service](http://coreos.com/rkt/docs/latest/subcommands/api-service.html)
 must be running on the node.
 
 ### Network Setup
```

Comment:
@yifan-gu I guess it's kubenet specific, not only rktnetes, see https://github.com/kubernetes/kubernetes/blame/v1.3.0-beta.1/pkg/kubelet/network/kubenet/kubenet_linux.go#L333, introduced in https://github.com/kubernetes/kubernetes/commit/5de7b561e350d43d39f1db00d8c5655dff165f0d